README.md

```diff
@@ -26,3 +26,12 @@ A custom scheduler to easily run tasks either in the back- or the foreground.
 
 * [UIView+Autolayout](https://github.com/novoda/ios-demos/blob/master/UIView%2BAutolayout.swift)
 extends UIView; Import this into your project to be able to use constraints programmatically in a straightforward way.
+
+## How to add a new project
+ - Pull from master and create a new branch with your project and/or changes
+ - If you are creating a new project make sure that XCode did not created a `.git` file inside of your folder. You can check that by going inside the project folder on terminal and entering `ls -la` if there is a `.git` file it will conflict with the git repository of the `ios-demos` repo. Delete it by entering `rm -rf .git` while being inside your project folder.
```

Comment:
I'd tweak:

```
 - If you are creating a new project, make sure that XCode did not create a `.git` file in your folder.
   You can check that by going inside the project folder on terminal and entering `ls -la`; if there is a `.git` file it will conflict with the git repository of the `ios-demos` repo.
   You can delete the unwanted file by entering `rm -rf .git` in the Terminal from inside your project folder.
```

I'm not sure what is a `.git` file, is it one of XCode's silly artifacts? Or is it a Git metadata folder?

Reply:
Xcode creates a lot of the times a GIT metadata folder when you create a new project so then you get a git inside a git and when you try to add the folder it adds it as a submodule and is a mess basically

Reply:
So if I understand it correctly, XCode initialises a Git repo in the current folder when you create a project, and that creates a `.git` metadata folder that you likely want to remove as the VCS root is at a higher level. Correct? If so change `file` with `folder` :)

Reply:
yes you understood correctly, I am changing now!